File: Symfony4/src/Entity/Disponibilite.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Serializer\Annotation\Groups;


/**
 * Disponibilite
 *
 * @ORM\Table(name="disponibilite", indexes={@ORM\Index(name="fk_disponibilite_helper_id", columns={"helperId"})})
 * @ORM\Entity(repositoryClass="App\Repository\DisponibiliteRepository")
 */
class Disponibilite
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @Groups("meet:search")
     * @ORM\Column(name="dateDeb", type="string", length=255, nullable=false)
     */
    private $datedeb;

    /**
     * @var string
     *
     * @Groups("meet:search")
     * @ORM\Column(name="dateFin", type="string", length=255, nullable=false)
     */
    private $datefin;

    /**
     * @var int
     *
     * @ORM\Column(name="etat", type="integer", nullable=false)
     */
    private $etat = '0';

    /**
     * @var \Users
     *
     * @ORM\ManyToOne(targetEntity="Users")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="helperId", referencedColumnName="Id")
     * })
     */
    private $helperid;

    public function __construct()
    {

    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDatedeb(): ?\DateTime
    {
        return new \DateTime($this->datedeb);
    }

    public function setDatedeb(\DateTime $datedeb): self
    {
        $this->datedeb = $datedeb->format("Y-m-d h:i:s");

        return $this;
    }

    public function getDatefin(): ?\DateTime
    {
        return new \DateTime($this->datefin);
    }

    public function setDatefin(\DateTime $datefin): self
    {
        $this->datefin = $datefin->format('Y-m-d h:i:s');

        return $this;
    }

    public function getEtat(): ?int
    {
        return $this->etat;
    }

    public function setEtat(int $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    public function getHelperid(): ?Users
    {
        return $this->helperid;
    }

    public function setHelperid(?Users $helperid): self
    {
        $this->helperid = $helperid;

        return $this;
    }


}

File: CodenameOne/src/com/mycompany/myapp/entities/Disponibilite.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author α Ω
 */
public class Disponibilite {
    
        private int id,etat, helperid;
    private String datedeb,dateFin;

    public Disponibilite() {
    }

    public Disponibilite(int id, int etat, int helperid, String datedeb, String dateFin) {
        this.id = id;
        this.etat = etat;
        this.helperid = helperid;
        this.datedeb = datedeb;
        this.dateFin = dateFin;
    }
    
    public Disponibilite(int id, int etat, String datedeb, String dateFin) {
        this.id = id;
        this.etat = etat;
        this.helperid = helperid;
        this.datedeb = datedeb;
        this.dateFin = dateFin;
    }

    public Disponibilite(int helperid, String datedeb, String dateFin) {
        this.helperid = helperid;
        this.datedeb = datedeb;
        this.dateFin = dateFin;
    }

    public int getId() {
        return id;
    }
    
    public void setId(int id) {
        this.id = id;
    }

    public int getEtat() {
        return etat;
    }

    public void setEtat(int etat) {
        this.etat = etat;
    }

    public int getHelperid() {
        return helperid;
    }

    public void setHelperid(int helperid) {
        this.helperid = helperid;
    }

    public String getDatedeb() {
        return datedeb;
    }

    public void setDatedeb(String datedeb) {
        this.datedeb = datedeb;
    }

    public String getDateFin() {
        return dateFin;
    }

    public void setDateFin(String dateFin) {
        this.dateFin = dateFin;
    }

    @Override
    public String toString() {
        return "disponiilite{" + "id=" + id + ", etat=" + etat + ", helperid=" + helperid + ", datedeb=" + datedeb + ", dateFin=" + dateFin + '}';
    }
    
    
    
    
    
}
